backend/src/providers/deployments/vercel.provider.ts

```
@@ -384,7 +384,7 @@ export class VercelProvider {
       };
     } catch (error) {
       if (axios.isAxiosError(error) && error.response?.status === 404) {
-        throw new AppError(`Deployment not found: ${deploymentId}`, 404, ERROR_CODES.NOT_FOUND);
+        throw new AppError(`Deployment not found: ${deploymentId}`, 404, ERROR_CODES.DEPLOYMENT_NOT_FOUND);
```

ESLint

[error] Replace ``Deployment·not·found:·${deploymentId}`,·404,·ERROR_CODES.DEPLOYMENT_NOT_FOUND` with `⏎··········`Deployment·not·found:·${deploymentId}`,⏎··········404,⏎··········ERROR_CODES.DEPLOYMENT_NOT_FOUND⏎········`

(prettier/prettier)